contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryBase2_3.sol

```diff
@@ -584,43 +621,60 @@ abstract contract AutomationRegistryBase2_3 is ConfirmedOwner {
     } else {
       gasWei = uint256(feedValue);
     }
-    (, feedValue, , timestamp, ) = i_linkNativeFeed.latestRoundData();
+    (, feedValue, , timestamp, ) = i_linkUSDFeed.latestRoundData();
     if (
       feedValue <= 0 || block.timestamp < timestamp || (staleFallback && stalenessSeconds < block.timestamp - timestamp)
     ) {
-      linkNative = s_fallbackLinkPrice;
+      linkUSD = s_fallbackLinkPrice;
+    } else {
+      linkUSD = uint256(feedValue);
+    }
+    (, feedValue, , timestamp, ) = i_nativeUSDFeed.latestRoundData();
+    if (
+      feedValue <= 0 || block.timestamp < timestamp || (staleFallback && stalenessSeconds < block.timestamp - timestamp)
```

Comment:
super nit, <= is 2 operations and a bit more expensive than <, so I guess feedValue < 1 does the same and a bit cheaper. Yes, it could sacrifice readability, so up to you.

Reply:
love it 🎉